mypy.ini

```diff
@@ -12,3 +12,6 @@ ignore_missing_imports = True
 
 [mypy-pytz.*]
 ignore_missing_imports = True
+
+[mypy-yaml.*]
+ignore_missing_imports = True
```

Comment:
pyyaml has a types package at https://pypi.org/project/types-PyYAML/

We can potentially include it in dev requirements?

Reply:
Thanks for sharing! Did not know about the package :) Will evaluate in the context of the optional dependency and add it maybe